### DISCUSSION TITLE
What's up with my profile?

### DISCUSSION THREAD

**Discussion by MaoJiayang** [community/community#51583] [GitHub Education]
![Screenshot_20230331_225935](https://user-images.githubusercontent.com/90536762/229157709-8768d175-340c-42c8-a895-96a97957faf2.jpg)
i have already completed all my profile and applied for 12 times!
can anyone tell me what's wrong with my profile?

**Comment by LukyynCZ** [comment]
I have same issue, I tried ISIC, my School ID and official document from school that I study there. I tried it for 28 times and still not getting aproved and I don't know what I am doing wron

**Comment by xycld** [comment]
same, If there is a workable solution, please reply me, thank you very much!

**Reply by xycld** [reply]
It's hard to see Chinese students being allowed to be certified, I've only been around one and I think maybe it's because the pre-processing model doesn't have Chinese

**Comment by jamesopppp** [comment]
me too !!!

**Comment by xycld** [comment]
https://kuizuo.cn/github-student-certification/#3修改-github-个人信息
I see there are a lot of Chinese students here, so maybe you can refer to this article, although it didn't do much for me

**Comment by MaoJiayang** [comment]
Thanks alot for help but that article also didn't do much for me.B4 i raise question here ive already read that article.Sent from my phone-------- Original message --------From: xy_cloud ***@***.***>Date: Mon, Apr 10, 2023, 20:04To: community/community ***@***.***>Cc: 茅嘉阳 ***@***.***>, Author ***@***.***>Subject: Re: [community/community] What's up with my profile? (Discussion #51583)
https://kuizuo.cn/github-student-certification/#3修改-github-个人信息
I see there are a lot of Chinese students here, so maybe you can refer to this article, although it didn't do much for me

**Comment by samus-aran** [comment]
Hi @MaoJiayang 

Here are some tips submitting your evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- **We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.**
- Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list. If you are submitting multiple evidence select the most preferable e.g. Student ID even if there is a Student ID + transcript + letterhead.  

Let us know how you get on 👍

**Comment by MaoJiayang** [comment]
thanks for your detailed explanation.ive successfully passed the verification this morning after about 28 tries.maybe it's because i changed my name in profile into english.Sent from my phone-------- Original message --------From: Ashley Nicolson ***@***.***>Date: Wed, Apr 12, 2023, 02:16To: community/community ***@***.***>Cc: 茅嘉阳 ***@***.***>, Mention ***@***.***>Subject: Re: [community/community] What's up with my profile? (Discussion #51583)
Hi @MaoJiayang
Here are some tips submitting your evidence:
If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list. If you are submitting multiple evidence select the most preferable e.g. Student ID even if there is a Student ID + transcript + letterhead.
Let us know how you get on 👍

**Reply by samus-aran** [reply]
Wonderful news @MaoJiayang ! 🎉  That's awesome to hear. Thank you for returning and providing an update ✨ Mark the most suitable answer so others having the same rejection reasons can see if it helps them too 👍 

What offer in the Student Developer Pack are you most excited to use first?! 🏅

**Comment by MaoJiayang** [comment]
也许你可以多试试，这个验证很考验坚持不懈。。。我申请了将近28次才成功。可能的要点有：你的名字要和证件统一证件分辨率不足改完profile之后重新登陆一下还没坚持到足够多的次数。。。希望你也能通过！Sent from my phone-------- Original message --------From: ZHANZIXIAN ***@***.***>Date: Mon, Apr 10, 2023, 17:18To: community/community ***@***.***>Cc: 茅嘉阳 ***@***.***>, Author ***@***.***>Subject: Re: [community/community] What's up with my profile? (Discussion #51583)
me too !!!

**Reply by zhanzixian** [reply]
Congratulations！

**Comment by whitewatercn** [comment]
可以参考我的这个帖子再逝一逝 https://xlog.whitewater.wang/Github_Student_Developer_Pack
You can refer to this post to try again https://xlog.whitewater.wang/Github_Student_Developer_Pack
